### PR TITLE
Rebuild for PyPy3.8 and PyPy3.9

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Retrying #84

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ requirements:
     - python
     - pywin32-ctypes  # [win]
     - secretstorage >=3.2  # [linux]
-    - importlib_metadata  # [py<310]
+    - importlib_metadata >=3.6  # [py<310]
```
Comment (xylar) [inline recipe/meta.yaml:32]: I'm fixing the constraint on `importlib_metadata` here rather than in #88

**Comment by xylar** [comment]
I have a repo patch ready to go in for the incorrect constraint on `importlib_metadata` in #87 as soon as this goes in.

**Comment by ccordoba12** [comment]
Looks good to me, thanks for your help with this @xylar!

**Comment by xylar** [comment]
Thank you, @ccordoba12!

**Pull request merged**